### PR TITLE
Patch nr1

### PR DESCRIPTION
**Pull request by xOBEMAx**
add  int a = 0;

INT@

### CODE REVIEW

**Comment by pcefcone** [comment]
Good job, you're fired

File: TMS_1/Program.cs [hunk 1]
```diff
@@ -1,2 +1,3 @@
 ﻿// See https://aka.ms/new-console-template for more information
 Console.WriteLine("Hello, World!");
```
Comment (pcefcone) [inline TMS_1/Program.cs:2]: its okay

**Review by pcefcone — COMMENTED** [review]
Ok

File: TMS_1/Program.cs (hunk 1 above)
Comment (xOBEMAx) [reply TMS_1/Program.cs:2]: lol

**Comment by pcefcone** [comment]
=)

**Pull request merged**